manifest.php

```diff
@@ -15,18 +15,15 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  * 
  * Copyright (c) 2016 (original work) Open Assessment Technologies SA;
- *               
- * 
  */
-use oat\ltiProctoring\controller\ProctoringTool;
+use oat\ltiProctoring\controller\DeliveryServer;
 use oat\ltiProctoring\controller\Monitor;
+use oat\ltiProctoring\controller\ProctoringTool;
 use oat\ltiProctoring\controller\Reporting;
-use oat\taoLti\models\classes\LtiRoles;
-use oat\tao\model\user\TaoRoles;
-use oat\ltiProctoring\scripts\install\SetupProctoringEventListeners;
-use oat\ltiProctoring\scripts\install\RegisterAuthProvider;
 use oat\ltiProctoring\scripts\install\RegisterServices;
```

Comment:
Please do not do formatting and other changes which are not related to fix. It makes review much harder. Do that it separate pull request.